resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -35,12 +35,14 @@ if ([string]::IsNullOrEmpty($env:TestRegistryPath)) {
     $global:ControlPanelAccessibilityRegistryPath= 'HKCU:\Control Panel\Accessibility\'
     $global:AudioRegistryPath = 'HKCU:\Software\Microsoft\Multimedia\Audio\'
     $global:PersonalizationRegistryPath = 'HKCU:\Software\Microsoft\Windows\CurrentVersion\Themes\Personalize\'
+    $global:NTAccessibilityRegistryPath = 'HKCU:\Software\Microsoft\Windows NT\CurrentVersion\Accessibility\'
+    $global:CursorIndicatorAccessibilityRegistryPath = $global:AccessibilityRegistryPath + '\CursorIndicator'

```

Comment:
I don't see much benefit to reusing an existing variable. I would just write it out explicitly. That way you don't run into issues with your tests having dependencies on a global variable that has not yet been defined.